src/complex/Pipeline/PipelineFilter.hpp

```diff
@@ -220,5 +226,6 @@ class COMPLEX_EXPORT PipelineFilter : public AbstractPipelineNode
   std::vector<complex::Error> m_Errors;
   std::vector<IFilter::PreflightValue> m_PreflightValues;
   std::vector<DataPath> m_CreatedPaths;
+  std::vector<DataPath> m_ModifiedPaths;
```

Comment:
I think we should store the whole struct in case we want to retrieve the type modification.